discovery/cmd/openstack-discoverer/main.go

```diff
@@ -123,13 +124,16 @@ func main() {
 		osClient.HTTPClient.Transport = httpTransportWithCA(log, openstackCertificateAuthorityFile)
 	}
 
+	osClient.HTTPClient = newHTTPClient()
```

Comment:
I believe this will break the ability to provide a CA cert for use in the HTTP transport (L123). We should also make sure to respect the timeout passed via flag and set in L121.

Reply:
Let me double check